docs/index.md

```diff
@@ -78,7 +78,7 @@ Our documentation is organized into the following categories:
   * [Development](#development-tools)
   * [Performance analysis](#performance-analysis)
   * [System](#system-tools)
-* [Hardware specifications](./reference/gpu-arch-specs.rst)
+* [Hardware specifications](./reference/accelerator-gpu-hardware-specs.rst)
```

Comment:
accelerator and gpu mean the same thing here, why double up?

Reply:
Mostly to reflect the title (h1) of the hardware specifications page.

We decided there is a non-trivial difference between *accelerators* (Instinct / CDNA) and *GPUs* (Radeon / RDNA) and the should docs reflect this.

Reply:
Sorry @saadrahim, I should have used `git mv`.

It looks like git was smart enough to preserve the blame this time. Or do you want me to revert?

I'll use `git mv` moving forward.

Reply:
Also please not that files should only be moved/renamed if absolutely necessary. Without an explicit redirect this invalidates bookmarks of users, and in specific circumstances can even break links in the documentation itself (though that shouldn't happen if the references have been set up properly)

Also right now the text in the table of contents and the overview in the index does not match the new title of the document ("Accelerator and GPU hardware specifications" vs "Hardware Specifications", this file and _toc.ylml.in)

Reply:
Thanks, that makes sense. Reverted the rename.

We want precise `h1`s. If that prevents the title from fitting in the TOC sidebar, we abbreviate it. From talks with @lpaoletti, we are okay with "Accelerator and GPU hardware specifications" as the `h1` and "Hardware specifications" in the sidebar.